src/html5-qrcode-scanner.ts

```diff
@@ -348,14 +363,27 @@ export class Html5QrcodeScanner {
         return this.html5Qrcode.getRunningTrackCapabilities();
     }
 
+    /**
+     * Returns the supported settings of the running video track.
```

Comment:
probably not accurate documentation, fix this here, in html5-qrcode, changelog and readme.